misc/inmanta-workon-register.sh

```diff
@@ -205,9 +204,13 @@ function __inmanta_workon_register_deactivate {
         unset -f inmanta >/dev/null 2>&1
         # no need to restore PS1 because virtualenv_deactivate already does that
 
+        if [ -n "${INMANTA_CONFIG_ENVIRONMENT}" ] ; then
+            unset INMANTA_CONFIG_ENVIRONMENT
```

Comment:
I actually think we should restore it if it was set before. Something like `PYTHONHOME` in `activate` scripts, but slightly more advanced.